spec/docker_registry2_spec.rb

```diff
@@ -80,7 +80,7 @@
       end
 
       it { expect { my_ubuntu_multiarch_manifest }.not_to raise_error }
-      it { expect(archs).to match_array(%w[amd64 amd64]) }
```

Comment:
I don't think that this test made sense as there should not be any image with the same architecture twice

Reply:
I didn't actually write the spec (and am appreciative to those who did), but I think you are right. How did it ever have both same arch?

Although, arguably, you _could_. The manifests in an index are in an array, so nothing is preventing duplication.